app/Http/Controllers/Api/V1/FeatureController.php

```diff
@@ -171,7 +170,7 @@ public function toggleByFeatureId(ToggleByFeatureId $request, int $featureId)
             throw new NotFoundException();
         }
 
-        $hasScope = 
+        $hasScope =
```

Comment:
Feature definition is enough for now. I have no plans to have those guard by scope for particular users. On or off is sufficient, given we have several envs